tracer/src/Datadog.Trace/Agent/AgentWriter.cs

```diff
@@ -447,6 +494,7 @@ SpanBuffer SwapBuffers()
             }
 
             // Add the current keep rate to the root span
+            var traceContext = spans.Array![spans.Offset].Context.TraceContext;
```

Comment:
Can we move this inside `if (IsLogLevelDebugEnabled)` where it's used below?

Reply:
Yep done!